runtime/common/src/tests/setup/mod.rs

```diff
@@ -42,8 +42,8 @@ pub fn charlie() -> AccountId {
 	AccountId::from(CHARLIE)
 }
 
-pub const PARA_A_USER_INITIAL_BALANCE: u128 = 500_000 * UNIT;
-pub const PARA_B_USER_INITIAL_BALANCE: u128 = 600_000 * UNIT;
+pub const PARA_A_USER_INITIAL_BALANCE: u128 = 500_000_000_000 * UNIT;
+pub const PARA_B_USER_INITIAL_BALANCE: u128 = 600_000_000_000 * UNIT;
```

Comment:
What do you mean?

Reply:
`UNIT` is already changed from 10^ 12 to 10^18 in `runtime/common/src/lib.rs`, we don't have to change anything here, same for other places

Reply:
This is leftover change since we previously seperate balance runtime impl PR and pallet migration PR.
Yes, this can be removed. But it will not cause any further problem since it is for avoiding existential deposit balance issue.